File: ReactNative/src/modules/signup/services/signup.service.ts
import axios from 'axios';
import { ISignupForm } from '../../../@types/signup-form';
import { SIGNUP_URL } from '../../common/utils/url';

export async function requestSignup(form: ISignupForm): Promise<void> {
  await axios.post(SIGNUP_URL, form);
}

File: flutter_todo/lib/src/modules/signup/services/signup_service.dart
import 'package:dio/dio.dart';
import 'package:flutter_todo/src/modules/common/utils/url.dart';
import 'package:flutter_todo/src/modules/signup/models/signup_model.dart';

Future<void> requestSignup(SignupModel formData) async => await Dio()
    .post(signupUrl, data: formData.toJson())
    .catchError((error) => throw error);
